ISISComputingGroup/EPICS-DbUnitChecker
# Ticket1491: Check for duplicate fields and duplicate PVs

Fixes https://github.com/ISISComputingGroup/IBEX/issues/1491

Depends on https://github.com/ISISComputingGroup/EPICS/pull/49

To test:
Run the test .bat file and confirm that the pvs in the test dbs pass/fail as expected

Note that when this is merged the Jenkins build will fail as it does not pass this test
